tla/consensus/MCccfraft.tla

```diff
@@ -92,20 +92,12 @@ MCSend(msg) ==
 MCInMaxSimultaneousCandidates(i) ==
     Cardinality({ s \in GetServerSetForIndex(i, commitIndex[i]) : state[s] = Candidate}) < 1
 
-\* Alternative to CCF!InitReconfigurationVars that sets the initial configuration to the first config defined in Configurations
-MCInitReconfigurationVars ==
-    /\ reconfigurationCount = 0
-    /\ removedFromConfiguration = {}
-    /\ configurations = [i \in Servers |-> 0 :> Configurations[1]]
-
 \* Alternative to CCF!Init that uses the above MCInitReconfigurationVars
 MCInit ==
-    /\ MCInitReconfigurationVars
+    /\ CCF!InitReconfigurationVars
```

Comment:
`MCccfraft` will not reach the first configuration with this approach. I think MC needs a separate init

Reply:
I've tried the change below, but this seems to cause the MC to blow past even the already higher number of distinct states (~8m) this PR generates compared to baseline (~2m), and it's still going.

```
1Configuration == <<{NodeOne, NodeTwo, NodeThree}>>
2Configurations == <<{NodeOne, NodeTwo}, {NodeTwo, NodeThree}>>
3Configurations == <<{NodeOne, NodeTwo}, {NodeTwo, NodeThree}, {NodeTwo}>>

[..,]

MCChangeConfigurationInt(i, newConfiguration) ==
    /\ reconfigurationCount < Len(Configurations)
    \* reconfigurationCount starts at 0, +1 to get the next configuration
    /\ newConfiguration = Configurations[reconfigurationCount+1]
    /\ CCF!ChangeConfigurationInt(i, newConfiguration)
```

Reply:
Restricting configurations to two nodes seems to help considerably.

Reply:
TLC exposes an "inspection hatch" in its JMX management interface with which you can observe the currently explored states.   That might give clues as to what is going on, i.e., on what dimension the state space is exploding.  I use VisualVM as a JMX frontend.